src/linter/root.go

```diff
@@ -902,19 +907,26 @@ func (d *rootWalker) handleClassDoc(doc classPHPDocParseResult, cl *meta.ClassIn
 	}
 }
 
-func (d *rootWalker) parsePHPDocVar(doc phpdoc.Comment) (typesMap types.Map) {
-	for _, part := range doc.Parsed {
-		part, ok := part.(*phpdoc.TypeVarCommentPart)
+func (d *rootWalker) parsePHPDocVar(doc phpdoc.Comment) (typesMap types.Map, deprecationInfo meta.DeprecationInfo) {
```

Comment:
`DeprecationInfo` should not be part of `parsePHPDocVar`, even the function name says that on parses `var` tag.

Reply:
Int this case the contex of Var - variable (property). See parseConstPHPDoc, parseClassPHPDoc: in all cases we are parse everything